README.md

```diff
@@ -2,10 +2,17 @@
 
 [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/performance/performance-ci?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=271&branchName=master)
 
-This repo contains benchmarks used for testing the performance of .NET Frameworks.
+This repo contains benchmarks used for testing the performance of all .NET Runtimes: .NET Core, Full .NET Framework, Mono and CoreRT.
 
-See the [Microbenchmarks Guide](./src/benchmarks/micro/README.md) for information on running our microbenchmarks.
-See the [Real-World Scenarios Guide](./src/benchmarks/real-world/JitBench/README.md) for information on running our real-world scenario benchmarks.
+We know that having the benchmarks in a separate repository might seem to be surprising. But the performance is vertical. .NET Runtime features affect base class library performance. Which at the end, affects the end user and provides the business value. This is why we keep all the benchmarks in a single place - to test the performance of the end product. For whatever .NET Runtime you want.
+
+## Documentation
+
+* [Prerequisites](./docs/prerequisites.md) (dotnet cli or Python 3)
+* [Microbenchmarks Guide](./src/benchmarks/micro/README.md) for information on running our microbenchmarks
+* [Real-World Scenarios Guide](./src/benchmarks/real-world/JitBench/README.md) for information on running our real-world scenario benchmarks
+
```

Comment:
JitBench happens to be the only real world scenario we have now but we'll have others in the not distant future. Perhaps there should be a readme in real-world and jitbench gets its own with specific info?

Reply:
I agree, but I think that such a change should be done when we change the JitBench/add new Scenarios.